src/resources.js

```diff
@@ -97,7 +97,7 @@ function getMetaFromResource(resource) {
   }
   return {
     reducer: 'object',
-    ...resource,
+    ...omit(resource, 'queries'),
```

Comment:
can u please explain why should it omit something? in resource u have data from api. It should not be omited